libbpf-tools/runqslower.c

```diff
@@ -33,18 +36,20 @@ const char *argp_program_bug_address =
 const char argp_program_doc[] =
 "Trace high run queue latency.\n"
 "\n"
-"USAGE: runqslower [--help] [-p PID] [-t TID] [-P] [min_us]\n"
+"USAGE: runqslower [--help] [-p PID] [-t TID] [-P] [min_us] [-c COMM]\n"
 "\n"
 "EXAMPLES:\n"
 "    runqslower         # trace latency higher than 10000 us (default)\n"
 "    runqslower 1000    # trace latency higher than 1000 us\n"
 "    runqslower -p 123  # trace pid 123\n"
 "    runqslower -t 123  # trace tid 123 (use for threads only)\n"
+"    runqslower -c bash # trace bash only (max comm name length is 15 chars)\n"
 "    runqslower -P      # also show previous task name and TID\n";
 
 static const struct argp_option opts[] = {
 	{ "pid", 'p', "PID", 0, "Process PID to trace", 0 },
 	{ "tid", 't', "TID", 0, "Thread TID to trace", 0 },
+	{ "comm",  'c', "COMM",  0, "Trace this comm only", 0 },
```

Comment:
If this patch is intended to "filter by process name prefix," it is not consistent with the option description.